sources/packages/web/src/types/contracts/DataTableContract.ts

```diff
@@ -101,3 +136,25 @@ export interface PageAndSortEvent {
   sortField: string;
   sortOrder: number;
 }
+
+/**
+ * Vuetify dataTable sort order.
+ */
+export enum DataTableSortByOrder {
+  DESC = "desc",
+  ASC = "asc",
+}
+
+/**
+ * DataTable options.
+ */
+export interface DataTableOptions {
+  itemsPerPage: number;
+  page: number;
+  sortBy: { key: string; order: DataTableSortByOrder }[] | [];
```

Comment:
Correct me if wrong, the `| []` is to accommodate the current sort order in place.

If so having a to-do here will be helpful to remove that.

Reply:
mm, I didn't understand. when there is no sort (sort icon not clicked) then sortBy: will be empty array

Reply:
I got it now.